oauth2_http/java/com/google/auth/oauth2/ExternalAccountSupplierContext.java

```diff
@@ -0,0 +1,78 @@
+package com.google.auth.oauth2;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import java.io.Serializable;
+
+/** Context object to pass relevant variables from external account credentials to suppliers. */
+public class ExternalAccountSupplierContext implements Serializable {
+
+  private static final long serialVersionUID = -7852130853542313494L;
+
+  private final String audience;
+  private final String subjectTokenType;
+
+  /** Internal constructor. See {@link ExternalAccountSupplierContext.Builder}. */
+  ExternalAccountSupplierContext(Builder builder) {
+    this.audience = builder.audience;
+    this.subjectTokenType = builder.subjectTokenType;
+  }
+
+  /**
+   * Gets the credentials expected audience.
```

Comment:
nit: Let's just use Returns instead of Gets everywhere

```suggestion
   * Returns the credentials' expected audience.
```